### PR TITLE
Removes Standard cyborg module.

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

Removes the Standard borgo module loadout.

## Why It's Good For The Game

Cyborg modules should be hyper-specialised to a single task. The Standard module breaks away from that design philsophy by making a single module that can do basically a bit of everything.

If a cyborg wants to be a jack-of-all-trades, they always have the option of getting a module reset to operate in a new area.

Cyborgs are encouraged to change modules as and when appropriate throughout the shift to deal with problems across the station depending on their own experience.

## Changelog
:cl:
del: NanoTrasen has stopped supporting Standard cyborg modules, citing spiralling costs with managing legacy hardware and ancient upgrades.
/:cl:

### CODE REVIEW

**Comment by brotherbeyondd** [comment]
People already think Peacekeeper means shitty secborg, please don't make them think this more.

**Comment by TheVekter** [comment]
HARM ALARM HARM ALARM

**Comment by anconfuzedrock** [comment]
why not just remove zipties from standard and remove peacekeeper borgs for being a shitty meme?

**Comment by ReplicantTerry** [comment]
> Cyborgs are encouraged to change modules as and when appropriate throughout the shift

"ok you know how to reset a cyborg right. cut only the starred wire"
snipsnipsnipsnipsnip
"fucks sake you unsynced me can you resync me"
"im still not synced"
"im still not synced"
"im still not synced"
continues, forever

**Comment by necromanceranne** [comment]
realistic-white-single-chicken-egg-isolated-white-background-template_153563-933

**Comment by IndieanaJones** [comment]
Turn secborgs back on.

**Comment by wesoda25** [comment]
I refuse to see a security associated borg with zipties.

**Comment by Timberpoes** [comment]
> 
> 
> > Cyborgs are encouraged to change modules as and when appropriate throughout the shift
> 
> "ok you know how to reset a cyborg right. cut only the starred wire"
> snipsnipsnipsnipsnip
> "fucks sake you unsynced me can you resync me"
> "im still not synced"
> "im still not synced"
> "im still not synced"
> continues, forever

I predicted this argument on the /tg/ coding Discord. I already came prepared to squash that argument like the insignificant bug it is.

#53277

**Comment by ishitbyabullet** [comment]
@Timberpoes i hate this pr simply because removing the standard module means you can't be the default cyborg without gimping yourself. Even if you removed 90% of their items, some stuff is better than literally being simplemob tier.

**Comment by ATH1909** [comment]
As someone who mains standard borg, I think you already know what my opinion of this PR is.

Somewhat relevant: https://github.com/TheSwain/Fulpstation/pull/247

> Standard borgs fill the niche of "I want to be able to deal with minor crime, but I don't want to be a secborg that can't do anything but validhunt", and I think that's a good niche to have filled. Playing secborg is really, really boring if the antagonists are all dead/not doing much, and standard borgs can do more stuff during that downtime, hence why I prefer playing them over security borgs.

Basically, playing standard borg is fun because you never feel _completely_ helpless, even though you can't do everything as well as a specialized borg can. Everyone's dying and on fire? You can extinguish them and then stabilize them with epinephrine, but you can't heal them as fast as a mediborg can. Someone just oxytank suicided in the HoPline? You can scrub away the gibs with soap and drag away their severed limbs ones by one, but you can't effortlessly clean the mess just by walking over it or pick up both arms and both legs in one trip like a janiborg can. Explosion? You can fix hull breaches and make iron walls, but you can't make walls as fast as or make windows like an engiborg can. Simplemob outbreak? You can bash them to death with your pickaxe and heal youself afterwards with your welder, but you can't kite them without taking damage like a miner borg can. The scientists need just a few more mats? You can find some ores with your scanner, mine them with your pickaxe, and then drag an ore box over them, but you can't kite lavaland mobs with a potent (on lavaland) ranged attack like, hold ores in a mining satchel like, or mine as fast as a minerborg can. Someone did something nice for you? You can hug them, but you can't make them a drink on the spot like a service borg can (we should definitely consider giving service borgs a hugging tool soon, though). Someone's being a shitter? You can flash and cuff them on the spot if they're not flashproof.

> **Cyborg modules should be hyper-specialised to a single task.** The Standard module breaks away from that design philsophy by making a single module that can do basically a bit of everything.

I guess what I'm trying to say here is that the standard borg's specialization is always being able to do _something_, and that makes it a unique and genuinely fun module to play as. A PK borg with cuffs wouldn't fill that "specialization", because the PK borg has a truly anemic number of tools at its disposal (in comparison to other modules) and can't really do much while not emagged (that can't be done by every other module) other than dispense cookies, validhunt, and be annoying.

You could also make an argument that the standard borg's specialization is "combat" (and I believe that I myself have said that before), but I think that that's because we don't really have a "true" combat specialist borg yet (other than secborgs, but they're gone, and PK borgs, but coders have been directly quoted as saying that they're intentionally a trap option and awful at filling the combat/"detaining criminals" niche), and so the standard borg, with its welder, pickaxe, and cuffs, is kind of the "combat" borg by default.

The only borg module that the standard borg really outclasses in its specialized role (nonlethal combat) is the PK borg, but that's mostly because PK borgs A. don't get cuffs and B. can't heal themselves (without the agonizingly slow self-repair upgrade). Were either of those problems to be fixed, the PK borg's ability to inject flashproof antags with pax would actually make it more than just a meme option (although pax not doing anything against flashes is still a big problem with the module's kit), although it still wouldn't scratch the same itch for me that playing a standard borg does, because the modules do two different things.

**Comment by ATH1909** [comment]
tldr;

Standard borgs have a unique and fun playstyle because they can do more than just validhunt antags, and giving a PK borg cuffs won't properly fill the niche left behind by the removal of standard borgs.

**Comment by Timberpoes** [comment]
Just. Change. Module.

The entire system is designed to encourage module changes to deal with pressing station needs. By default the module reset wire is noted on the hacking interface, with #53277 going a step further to make it even more obvious. Module changes are intended to be easy.

Module selection is not permanent. Get a module reset. With the above mentioned PR, any chimp with a wirecutter can do it. Roboticists already see all your wires even without that above tweak.

**Comment by Timberpoes** [comment]
Side note that, yes, it makes players who were formally standard borgs a lot more reliant on roboticists or other crew members in genreal if they wish to do a bit of everything across the station. This is a good thing. Roboticists have little to do with cyborgs in a given round beyond building them and that one time they come for their single round of upgrades or a battery upgrade. Shifting some work back to them can, in my eyes, only be an improvement.

**Comment by ishitbyabullet** [comment]
> The entire system is designed to encourage module changes to deal with pressing station needs. By default the module reset wire is noted on the hacking interface, with #53277 going a step further to make it even more obvious. Module changes are intended to be easy.

**IT'S NOT MERGED YET. I REPEAT - IT IS NOT MERGED YET. WHAT HAPPENS IF THIS IS MERGED YET NOT THAT?** You're going to look like a backstabbing liar if you tell us to deal with it in a way that isn't actually possible.

**Comment by Timberpoes** [comment]
> 
> 
> > The entire system is designed to encourage module changes to deal with pressing station needs. By default the module reset wire is noted on the hacking interface, with #53277 going a step further to make it even more obvious. Module changes are intended to be easy.
> 
> **IT'S NOT MERGED YET. I REPEAT - IT IS NOT MERGED YET. WHAT HAPPENS IF THIS IS MERGED YET NOT THAT?** You're going to look like a backstabbing liar if you tell us to deal with it in a way that isn't actually possible.

All that PR adds is the "Reset Module" tag to the reset module wire. It already has the star flavour text on live, which if the braying masses are to be believed just isn't good enough. However, it's literally already there. You just need to not be a smooth brain and explain it properly. "When you're hacking me, there'll be a section that tells you the status of my various functions at the bottom." "There it'll mention a wire with a star above it. Cut and mend that wire!"

It's already on live. The PR above just makes it super dummy thicc obvious.

**Comment by ATH1909** [comment]
> You just need to not be a smooth brain and explain it properly. "When you're hacking me, there'll be a section that tells you the status of my various functions at the bottom." "There it'll mention a wire with a star above it. Cut and mend that wire!"

I already tell roboticists literally exactly that, and they either cut every wire anyway or take a full minute to try to find that sentence about the starred wire in the wire interface. I'm not joking, that literally happens 50% of the time I try to get a module change.

You can see why I'm hesitant to trust "any chimp with a wirecutter" (and a screwdriver, and a crowbar if the module you chose doesn't have one) to change my module in anything resembling a timely manner, even if I say what they need to do 3 times in a row, yes?

**Comment by Timberpoes** [comment]
> 
> 
> > You just need to not be a smooth brain and explain it properly. "When you're hacking me, there'll be a section that tells you the status of my various functions at the bottom." "There it'll mention a wire with a star above it. Cut and mend that wire!"
> 
> I already tell roboticists literally exactly that, and they either cut (and maybe mend) every wire anyway or take a full minute to try to find that sentence about the starred wire in the wire interface. I'm not joking, that literally happens 50% of the time I try to get a module change.
> 
> You can see why I'm hesitant to trust "any chimp with a wirecutter" (and a screwdriver, and a crowbar if the module you chose doesn't have one) to change my module in anything resembling a timely manner, even if I say what they need to do 3 times in a row, yes?

As we hashed out on Discord, that sounds suspiciously like a you problem.

The Standard cyborg module does not (or, should I say, does but should not) exist as the powergamer's choice so they can do everyone's job simultaneously for them.

If a *roboticist* can't cut your module reset wire, they're literally fucking with you. Because they see your wire assignments thanks to their skillchip. If anyone else can't cut it, they're either fucking with you or you're asking someone not skilled in robotics to do a robotics procedure. Seems about right that it could fail half the time.

The entire argument you put forward goes against the spirit of cyborgs and cyborg modules. You're a subservient role to the human crew of the station. You have a radio. If the perp is getting away, tell sec where they went. If the hull is breached and atmos is leaking, call engineering. If people are crit, call the paramedics.

And then, depending on your module, maybe you can do one of those jobs yourself. Then if the others still need doing, get a module change and revive the deaddie. Go engy and fill the breach then restore appropriate atmos. Go janiborg and clean everything up. Go peacekeeper and scream HUMAN HARM at every greytider in a gas mask until you find the bomber. But not all of them at once.

**Comment by optimumtact** [comment]
I once saw ATHATH drag a mobile turret  around to valid hunt as a standard borg so frankly I don't really have any sympathy for the standard borg module.

**Comment by LemonInTheDark** [comment]
image

**Comment by ATH1909** [comment]
> I once saw ATHATH drag a mobile turret around to valid hunt as a standard borg so frankly I don't really have any sympathy for the standard borg module.

I did that ONCE on /tg/ to test it, geeze.

**Comment by optimumtact** [comment]
let this be a lesson to you that your actions have consequences

**Review by Mothblocks — DISMISSED** [review]

**Comment by ATH1909** [comment]
If anyone's curious, the results of the testing here and on fulp were that it's about as effective as beepsky, because you need to click two things instead of one to set it up.

And to be clear, I don't drag a secbot behind me 24/7 as a standard borg; I only grab one if ops declare war or if I'm facing a flashproof target.

**Comment by optimumtact** [comment]
Anyway, Timberpoes has outlined the main thrust of the reasons behind this but just to add weight to the comments

A) yes, borgs are intended to be hyperspecialised ,and module switching *is* something they should be doing with the roboticist.

B) after the secborg crap, I have zero interest in borgs that have any kind of valid hunt/security functionality. That's why the peacekeeper is setup to prevent harm, but not be easily able to bring people in. It leads to these awful debates about harm, asimov and robots not following laws.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -248,28 +248,6 @@
 		return FALSE
 	return TRUE
 
-/obj/item/robot_module/standard
```
Comment (ATH1909) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:251]: could you make them admin-only instead of fully removing them, like secborgs?

wait a minute, don't we have a config option to disable standard borgs?

Comment (ATH1909) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:251]:
```
	if(!CONFIG_GET(flag/disable_peaceborg))
		modulelist["Peacekeeper"] = /obj/item/robot_module/peacekeeper
	if(!CONFIG_GET(flag/disable_secborg))
		modulelist["Security"] = /obj/item/robot_module/security
```

Oh, we have a config option to disable secborgs and a config option to disable PK borgs, but no config option to disable standard borgs yet.

Maybe you could add one?

Comment (Jared-Fogle) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:251]: We're not having a config option for this when nobody is going to maintain it.

**Comment by PKPenguin321** [comment]
giving PK borgs zip ties is a bad idea and them not having any kind of cuffs was one of the vital conditions that allowed their original creator to get them into the game in the first place

**Comment by ATH1909** [comment]
image

**Comment by Jared-Fogle** [comment]
I think I would prefer if killing the standard borg and zipties were two different PRs so both can be discussed on their own merit, and because one can exist without the other.

Personally, I'd prefer *this* PR to just be standard borg removal since that seems to be the topic of discussion.

**Comment by zxaber** [comment]
The only real argument for borg cuffs sticking around that I know of is to keep the line held down outside a robo factory. To that end, locking the cuffs on PK to be behind the emag/hack would be fine.

**Comment by PKPenguin321** [comment]
>The only real argument for borg cuffs sticking around that I know of is to keep the line held down outside a robo factory. To that end, locking the cuffs on PK to be behind the emag/hack would be fine.

Would be very strong for emag, don't really like that either
I do think malf AI should be scarier though so I agree on that end

**Comment by IndieanaJones** [comment]
Never thought I'd see the day where of all the borgs to kick the can, it's the damn standard borg who's getting it, just seems very bizarre.  I mean, if you remove the cuffs, I don't really see the issue of keeping it, maybe reduce its welder to an emergency one as well.

**Comment by carshalash** [comment]
Love you timber but i'm gonna have to call this cringe, robotocists are some of the dumbest and most selfish players to exist and even snipping the wire tends to be too much work for them.

**Comment by ATH1909** [comment]
> Never thought I'd see the day where of all the borgs to kick the can, it's the damn standard borg who's getting it, just seems very bizarre. I mean, if you remove the cuffs, I don't really see the issue of keeping it, maybe reduce its welder to an emergency one as well. Even then, I don't often see it used, since its outclassed as the gamerborg in a number of respects (unless emagged, because that e-sword is really something).

Tbh, even as a standard borg main, I feel like the esword is overrated. Yeah, the esword wins a fight in four clicks, but the gamer arm wins a fight in one.

**Comment by wesoda25** [comment]
Yeah it’s almost as if we should nerf stun arm.

**Comment by CRITAWAKETS** [comment]
Wow, its almost like every borg is hyperspecialized BECAUSE the standard borg exists in the first place and that removing the literal core of cyborgs could cause issues. Also i thought removing unnessesary tedious stuff is a good thing.

Basically what im telling you is that i removed most of engineering's engines and yet i still think your removal is something that only some madman overdosing on crank could have the idea for.

**Comment by necromanceranne** [comment]
How borgs have dodged the stun phasing time and again is incredible. Even removing borg spin was a real uphill battle as I recall.

**Comment by CRITAWAKETS** [comment]
Reminder borgs still can be stunned by flashes and being grazed by EMPs. I get EMPs but why is flash stunning still a thing both on borgs and humans.

**Comment by GoldenAlpharex** [comment]
I have to admit, I too think that nerfing is a better way to go here, the literal reason behind having the standard borg is to be like "uh I'm new to this and I don't know what to choose" or to just be a weak jack-of-all-trades to have the other modules make sense as being specialized modules. While I do agree that we should enforce the changing modules bit, as it currently stands right now, having to rely on Roboticists that are usually too busy either augmenting people (or getting augmented), or making their gamer mechs, there's very little time frames during which Roboticists aren't busy and therefore only really nice Roboticist players will actually go through the hassle of reseting the cyborg's module. 

Another reason I'm strongly against this PR is that I feel like the Removal Culture there currently is on /tg/ is simply not beneficial to the game. We shouldn't be removing things that have been there for years, we should instead find ways to balance them with the new challenges there is nowadays.

Furthermore, I deeply feel, no offense intended, that the coder perspective is often a disfigured version of the player perspective. There's so many changes happening lately that just piss off the majority of the codebase, and I feel like honestly there should be more attention being paid to the desires of the community, and not just of the coders (and headcoders, obviously).

I'm not saying this to trash on anyone, I'm just trying to be constructive and making it so that the "big" PRs that we see nowadays aren't just deletions, because people don't want to put in the effort of just re-balancing things.

**Comment by Jared-Fogle** [comment]
> We shouldn't be removing things that have been there for years, we should instead find ways to balance them with the new challenges there is nowadays.

We don't need to keep every feature added from old coders whose basis is usually just "I thought it would be cool". 

Consider the OP's intentions with removing the standard borg.

> Cyborg modules should be **hyper-specialised to a single task.**

How exactly do you intend to "balance" the standard module into being hyper-specialized to a single task, which is the exact *opposite* of what it's supposed to be?

**Comment by GoldenAlpharex** [comment]
> > We shouldn't be removing things that have been there for years, we should instead find ways to balance them with the new challenges there is nowadays.
> 
> We don't need to keep every feature added from old coders whose basis is usually just "I thought it would be cool".
> 
> Consider the OP's intentions with removing the standard borg.

Yeah, but has anyone actually considered changing what's considered "OP" from them instead of just straight up removing them?

> > Cyborg modules should be **hyper-specialised to a single task.**
> 
> How exactly do you intend to "balance" the standard module into being hyper-specialized to a single task, which is the exact _opposite_ of what it's supposed to be?

Well, what can I say, I just don't think that the standard borg is used often enough to just replace every other cyborg to warrant the removal of them. The whole reason this kind of PR exists is because one person saw someone using something that's been in-game forever and that they probably used themselves, but then somebody else used it against them and suddenly it's the end of the world. Standard borgs are meant to be pretty much garbage at everything they do, but at least they can do it. It's literally the "Assistant" Cyborg, and I don't see how having the Standard module causes other cyborgs to feel useless. The simple fix to this is to just remove the zipties and to change the welder to be an emergency one, and congratulations, it's a worse version of every other cyborg, suited for basically no situation, always in need of help, but always there to at least do some kind of "first-aid" helping the station.

Now, I want to ask you all a question, because I'm genuinely curious: What exactly do we gain from removing the Standard Cyborg?

Does it make you feel better that the janitorborg is now the only one that can clean? That the Engiborg is the only one that can fix walls?

To me, it just feels like a matter of like "It makes me feel better about cyborgs knowing that standard borgs don't exist anymore."

If you've got an actual solid reason, I'm definitely down to hear it, and I'm up to switching sides. It's just that right now, I don't agree with this change, because I feel like the basis of it are very weak.

**Comment by Jared-Fogle** [comment]
> Yeah, but has anyone actually considered changing what's considered "OP" from them instead of just straight up removing them?

OP is in original poster, silly.

> The whole reason this kind of PR exists is because one person saw someone using something that's been in-game forever and that they probably used themselves, but then somebody else used it against them and suddenly it's the end of the world.

What suggested that from the post body?

> The simple fix to this is to just remove the zipties and to change the welder to be an emergency one, and congratulations, **it's a worse version of every other cyborg, suited for basically no situation**

I suggest reading the original poster's intentions again. I singled it out specifically in the post you are quoting. This isn't a good thing. The discussion isn't about if standard borgs are *overpowered*, it's about whether they are well designed. Something can be badly designed and not necessarily be under/overpowered.

> What exactly do we gain from removing the Standard Cyborg?

A more consistent design philosophy. 

> Does it make you feel better that the janitorborg is now the only one that can clean? That the Engiborg is the only one that can fix walls?

Yes.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Just going to repeat myself so that it stands out as a blocker--please turn this PR into only about standard borg removal, and make a new PR for giving zipties to peacekeeper borgs.

**Comment by Timberpoes** [comment]
Will do. I just woke up, but I'll atomise the change and PR it before I start work.

**Comment by IndieanaJones** [comment]
Why don't you make it a config option and let the admins decide?

**Comment by Jared-Fogle** [comment]
> Why don't you make it a config option and let the admins decide?

I answered this above.

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (IndieanaJones) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:251]: What exactly needs to be maintained?  I'll maintain it if necessary.

**Review by Mothblocks — APPROVED** [review]

**Comment by Timberpoes** [comment]
I will leave the following in response to people saying the standard module is the "begginner borg that everyone uses because it has no responsibilities":

We already have a cyborg shell that is dedicated to vibing around the station and getting to grips with how cyborgs work without being a third cog in the silicon machine.

The Butler model aka the Service Borg. It has no vital functions at all and exists to vibe across the station spraypainting obscenities, playing music, dispensing lollipops and mixing drinks.

It is the perfect module to figure out how laws work and your role on the station for a new cyborg player because it comes pre-loaded with zero expectations and a number of quirky, low-impact modules that allow you to safely vibe around the station, getting to know the ropes in a low pressure manner.

**Comment by IndieanaJones** [comment]
I still think this isn't the answer and a repurposing of the module would be better, but I guess you already won by virtue of maintainer approval, so I guess even trying to argue is pointless.

**Comment by Jared-Fogle** [comment]
Public disapproval isn't worthless to me, I'm intentionally leaving this PR open up for a few days to get more feedback on why the module should continue to exist. 

Precedent is not to leave in things we consider badly designed until a replacement is made. If you'd like a replacement, I definitely recommend making your own pull request. If you do so, make sure you consider the points made in why this is considered good for the game--cyborgs should be specialized. What is not obvious to me is how to make a "standard" module that is also specialized.

**Comment by IndieanaJones** [comment]
Well, just to throw forth some of my thoughts on the matter, I always thought the standard cyborg to mirror the assistant role, which essentially is a jack of all trades.  However, this borg really isn't since it has:

- 2 medical items, one checks health, another just saves people from dying in crit
- 6 engineering items, which includes the industrial welder, giving it the combat sustain of the engiborg as well
- 1 fire extinguisher, which is something every borg has
- 2 very lousy mining items that are publicly available
- soap
- hugging module
- flash
- zipties, which is its only unique item

As you can see, this heavily is skewed in terms of engineering items, which make up a majority of the kit.  It can be a very shitty miner, medic, or janitor, but it intrudes mostly on the engineer's domain.  To be honest, the whole specialization thing doesn't even make sense because this borg can barely do the jobs of 3/4's of what makes up its kit.  It essentially is good at two things:
- some engineering
- validhunting (welder + pickaxe + zipties + flash)
So even if you believe that a jack of all trades is bad, this borg isn't even filling that niche, and a more proper argument against the standard borg would be that you're removing a worse engieborg would gets validhunting tools as a tradeoff.  Now, I feel like the proper solution is instead of full-blown removal would be just to modify the kit to really highlight the ASSISTANT part of it, perhaps build a kit all around helping humans do general tasks (faster movespeed when being ridden to escort humans, stuff like that), as removing an artifact like this entirely seems very jarring.

I'm also writing this at like 5 AM, I think what I just said actually highlights why you may have a valid case in removing it, but what I guess I want to say is the concept of an "assistant" type cyborg isn't a bad one, its just poorly implemented as is currently but it could be fixed.  I would also say I'm just annoyed that this PR doesn't come with a rework of it is because this isn't an urgent removal and having it in the game a bit longer isn't harming anyone, but removing it without a replacement right now is just serving to upset people.

**Comment by Jared-Fogle** [comment]
> I always thought the standard cyborg to mirror the assistant role, which essentially is a jack of all trades. 

Again, the problem is that we do not believe a "jack of all trades" borg is good design.

I see your point of it stepping on the engi-borgs toes, but even if it was a jack of all trades, master of none, our problem is that we believe cyborg modules should be specialized.

>  I would also say I'm just annoyed that this PR doesn't come with a rework of it is because this isn't an urgent removal and having it in the game a bit longer isn't harming anyone, but removing it without a replacement right now is just serving to upset people.

As I said before, it's not obvious to me how you would make the idea of a "standard cyborg" specialized. I would like to see if someone can make this happen, though.

**Comment by JenqaDev** [comment]
image
the community doesn't want this removal, let's see how fast it gets merged...

**Comment by Jared-Fogle** [comment]
The community also isn't doing the best job explaining why it shouldn't be merged.

**Comment by JenqaDev** [comment]
fine, most of the comments are against the removal of standard cyborgs then. Taking them away is going to take part of cyborg fun
please we don't have hands we can't assemble anything on our own! let us have some fun with retarded standards roaming the station

**Comment by ATH1909** [comment]
> fine, most of the comments are against the removal of standard cyborgs then. Taking them away is going to take part of cyborg fun
> please we don't have hands we can't assemble anything on our own! let us have some fun with retarded standards roaming the station

It's not the quantity of comments that matters; it's persuasiveness of the arguments within them.

**Comment by msgerbs** [comment]
> 
> 
> The community also isn't doing the best job explaining why it shouldn't be merged. It's hard for me to have a productive conversation with an emoji.

You're not doing the best job explaining why it should be. Please explain _why_ a jack-of-all-trades borg is a bad idea, in more words than "we, the all-knowing maintainers that know better than you, think so".

**Comment by Jared-Fogle** [comment]
> Please explain why a jack-of-all-trades borg is a bad idea

From my point of view, the biggest problem is lessening the importance of going to the roboticist for module resets, thus decreasing player interaction.

This is mentioned in the OP as well:

> Cyborgs are encouraged to change modules as and when appropriate throughout the shift to deal with problems across the station depending on their own experience.

---

> It is literally in the template that you have to explain why something is good for the game, it should be on you to explain why it should be removed, not us to explain why it should be kept.

This is true. The reason why this seems to be the expectation is that I *was* convinced by the "why this is good for the game".

**Comment by IndieanaJones** [comment]
Also, I'm going to be completely honest, I've played a lot of cyborg in the last two years, I don't think I've ever asked for a module swap unless someone saw me with no module and law 2'd me to go engiborg to do some mundane shit for them and I wanted to do something else instead or people needed an engiborg.  If they really need a cyborg to do a certain job, we certainly have enough dead bodies and brains to go around, you can just make another one.  Considering these are all jobs are shared with a crewmember as well, module swapping is never really necessary at all.

**Comment by Tharcoonvagh** [comment]
I approve of this change, no more standard borgs dragging hacked secbots

**Comment by ATH1909** [comment]
If this had been merged a few hours earlier, I'd have posted My Way here. But it wasn't, so I won't.

You gave me some fun memories, standard module, and I'll always remember you for that. Goodbye.

**Comment by boodaliboo** [comment]
> > Please explain why a jack-of-all-trades borg is a bad idea
> 
> From my point of view, the biggest problem is lessening the importance of going to the roboticist for module resets, thus decreasing player interaction.
> 
> This is mentioned in the OP as well:
> 
> > Cyborgs are encouraged to change modules as and when appropriate throughout the shift to deal with problems across the station depending on their own experience.
> 
> > It is literally in the template that you have to explain why something is good for the game, it should be on you to explain why it should be removed, not us to explain why it should be kept.
> 
> This is true. The reason why this seems to be the expectation is that I _was_ convinced by the "why this is good for the game".

who the fuck cares stopping making the game unfun

**Comment by kriskog** [comment]
Good, only 5 to go now :+1:
